src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo | undefined,
+  setInicializationModal: (inicializationModal: boolean) => void,
+  setTodoModal: (selectedTodo: Todo | undefined) => void,
```

Comment:
It's better to use null instead of undefined. `null` is a value that can be assigned to a variable, whereas undefined means that a variable has not been assigned a value. 
```suggestion
  setTodoModal: (selectedTodo: Todo | null) => void,
```